gitlab/resource_gitlab_topic.go

```diff
@@ -0,0 +1,153 @@
+package gitlab
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabTopic() *schema.Resource {
```

Comment:
We've merged a PR which changes how resources are registered. Please look at another resource and use the same pattern. It's a fairly easy change :)